src/functions-reference/transform_functions.qmd

```diff
@@ -0,0 +1,537 @@
+{{< since 2.37 >}}
+
+<!-- vectors; unit_vectors_jacobian; (vectors y); -->
+\index{{\tt \bfseries unit\_vectors\_jacobian }!{\tt (vectors y): vectors}|hyperpage}
+
+`vectors` **`unit_vectors_jacobian`**`(vectors y)`<br>\newline
+Takes a free vector `y` and returns a vector with Euclidian length `1`.
+This function rejects if given a vector of all zeros.
+
+This function also increments the `target` variable
+with the corresponding change of variables adjustment.
+{{< since 2.37 >}}
+
+<!-- vectors; unit_vectors_unconstrain; (vectors x); -->
+\index{{\tt \bfseries unit\_vectors\_unconstrain }!{\tt (vectors x): vectors}|hyperpage}
+
+`vectors` **`unit_vectors_unconstrain`**`(vectors x)`<br>\newline
+Takes a vector `x` of unit length and returns the corresponding free vector.
+{{< since 2.37 >}}
+
+## Transforms for constrained matrices
+
+Similarly to the above, `vectors` means a `vector` or array thereof,
+and `matrices` means a `matrix` or array thereof.
+
+### Cholesky factors of correlation matrices
+
+
+These functions perform the mathematical transform described in the
+[Cholesky Factors of Correlation Matrices section](https://mc-stan.org/docs/reference-manual/transforms.html#cholesky-factors-of-correlation-matrices).
+
+<!-- matrices; cholesky_factor_corr_constrain; (vectors y, int K); -->
+\index{{\tt \bfseries cholesky\_factor\_corr\_constrain }!{\tt (vectors y, int K): matrices}|hyperpage}
+
+`matrices` **`cholesky_factor_corr_constrain`**`(vectors y, int K)`<br>\newline
+Takes a vector `y` and integer `K`, where `length(y) == choose(K, 2)`, and returns
+a `K` by `K` Cholesky factor of a correlation matrix.
+{{< since 2.37 >}}
+
+<!-- matrices; cholesky_factor_corr_jacobian; (vectors y, int K); -->
+\index{{\tt \bfseries cholesky\_factor\_corr\_jacobian }!{\tt (vectors y, int K): matrices}|hyperpage}
+Takes a vector `y` and integer `K`, where `length(y) == choose(K, 2)`, and returns
+a `K` by `K` Cholesky factor of a correlation matrix.
+
+This function also increments the `target` variable
+with the corresponding change of variables adjustment.
+`matrices` **`cholesky_factor_corr_jacobian`**`(vectors y, int K)`<br>\newline
+
+{{< since 2.37 >}}
+
+<!-- vectors; cholesky_factor_corr_unconstrain; (matrices x); -->
+\index{{\tt \bfseries cholesky\_factor\_corr\_unconstrain }!{\tt (matrices x): vectors}|hyperpage}
+
+`vectors` **`cholesky_factor_corr_unconstrain`**`(matrices x)`<br>\newline
+Takes `x`, a lower triangular square ($K$ by $K$) matrix which is the Cholesky factor of a correlation matrix,
+and returns a corresponding free vector of length $K\choose 2$.
+{{< since 2.37 >}}
+
+### Cholesky factors of covariance matrices
+
+These functions perform the mathematical transform described in the
+[Cholesky Factors of Covariance Matrices section](https://mc-stan.org/docs/reference-manual/transforms.html#cholesky-factors-of-covariance-matrices).
+
+<!-- matrices; cholesky_factor_cov_constrain; (vectors y, int M, int N); -->
```

Comment:
Same comments here as for correlation matrix---have to say what a Cholesky factor of a covariance matrix is!  The other way to say this is that it's a matrix $L$ such that $L \cdot L^\top$ is positive definite.  As usual, this can wind up being positive semi-definite due to numerical rounding.